### PR TITLE
UX: Hide tags section in sidebar when user has no visible tags

### PR DESCRIPTION
**Pull request by tgxworld**
Also hides the tags configuration for sidebar under user preferences

Internal ref: /t/73500

### CODE REVIEW

**Comment by tgxworld** [comment]
@lis2 Can you review this for me too? thanks!

File: app/serializers/concerns/user_sidebar_tags_mixin.rb [hunk 1]
```diff
@@ -2,10 +2,20 @@
...
 
   def include_sidebar_tags?
     SiteSetting.enable_experimental_sidebar_hamburger && SiteSetting.tagging_enabled
   end
+
+  def display_sidebar_tags_section
+    DiscourseTagging.filter_visible(Tag, scope).exists?
+  end
+
+  def include_display_sidebar_tags_section?
```
Comment (pmusaraj) [inline app/serializers/concerns/user_sidebar_tags_mixin.rb:18]: This is the same as `include_sidebar_tags?` above, right? I would personally go with the initial name, i.e. I would drop the `_section` suffix here.

**Comment by tgxworld** [comment]
@pmusaraj Thank you for reviewing :) I've updated the PR per your review comments.

**Review by pmusaraj — APPROVED** [review]

**Comment by tgxworld** [comment]
Thank you @pmusaraj

**Pull request merged**